test/ccapi/unittest_ccapi.cpp

```diff
@@ -201,7 +201,7 @@ TEST(nntrainer_ccapi, train_with_config_01_p) {
   EXPECT_NO_THROW(model->train());
 
   EXPECT_FLOAT_EQ(model->getTrainingLoss(), 4.434051);
-  EXPECT_FLOAT_EQ(model->getValidationLoss(), 2.9646113);
+  EXPECT_NEAR(model->getValidationLoss(), 2.9646113, 1.0e-5);
```

Comment:
how about reducing tolerance only for i586?

Reply:
Done